Models/Events/MessageEventModel.cs

```diff
@@ -8,14 +8,34 @@ namespace Theorem.Models.Events
 {
     public class MessageEventModel : EventModel
     {
+        private float threadId, timeSent;
+
         [JsonProperty("text")]
         public string Text { get; set; }
         
         [JsonProperty("ts")]
-        [JsonConverter(typeof(UnixDateTimeConverter))]
-        public DateTime TimeSent { get; set; }
+        public string TimeSentNative { get; set; }
```

Comment:
If we're changing our logic here to store Slack's native `TimeSent`, we should update the migration to drop the old `TimeSent` column, and rename `TimeSentNative` to `SlackTimeSent` to match the convention in `EventModel`.

`ThreadIDNative` -> `SlackThreadId`

Does anyone actually care about the `ThreadID` DateTime value? (`ThreadIDNative`). Is this any different than `TimeSent` with respect to Date+Time ? If not, we should remove `ThreadID`.

`TeamID` -> `TeamId` or `SlackTeamId` if this is a Slack identifier.

Reply:
Also wonder if it's worth storing a converted `DateTime` object on insertion to make querying easier (DateTime is easier to deal with in Linq than a string). We should probably have a way to do this, especially if we're not storing the `TimeSent` value as a number for comparison queries (lt, gt, etc).